rewrite-java/src/main/java/org/openrewrite/java/internal/TypesInUse.java

```diff
@@ -88,6 +88,15 @@ public J.ClassDeclaration visitClassDeclaration(J.ClassDeclaration c, Integer p)
             return c;
         }
 
+        @Override
+        public J.FieldAccess visitFieldAccess(J.FieldAccess fieldAccess, Integer integer) {
+            if ("class".equals(fieldAccess.getSimpleName()) && fieldAccess.getTarget() instanceof J.FieldAccess) {
+                J.FieldAccess target = (J.FieldAccess) fieldAccess.getTarget();
+                types.add(JavaType.buildType(target.toString()));
+            }
```

Comment:
This might not be necessary anyone if we manage to resolve the underlying missing type issue. Best to check that before a merge if we make that part of this PR.